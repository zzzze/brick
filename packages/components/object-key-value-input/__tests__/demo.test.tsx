import demoTest from '@brick/tools/tests/demoTest'

demoTest('components/object-key-value-input')
