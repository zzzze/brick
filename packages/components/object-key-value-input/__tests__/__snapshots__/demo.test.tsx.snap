// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./packages/components/object-key-value-input/demo/basic.md correctly 1`] = `
<div>
  <button
    data-testid="add-item"
  >
    add
  </button>
</div>
`;

exports[`renders ./packages/components/object-key-value-input/demo/with-props.md correctly 1`] = `
<div>
  <button
    data-testid="add-item"
  >
    add
  </button>
  <div>
    <input
      name="0-label"
      type="text"
      value="foo"
    />
    <input
      name="0-value"
      type="text"
      value="bar"
    />
    <button
      data-index="0"
      data-testid="remove-btn-0"
    >
      delete
    </button>
  </div>
  <div>
    <input
      name="1-label"
      type="text"
      value="baz"
    />
    <input
      name="1-value"
      type="text"
      value="123"
    />
    <button
      data-index="1"
      data-testid="remove-btn-1"
    >
      delete
    </button>
  </div>
</div>
`;
